llvm/test/Bitcode/compatibility.ll

```diff
@@ -564,6 +564,10 @@ declare riscv_vls_cc(32768) void @riscv_vls_cc_32768()
 ; CHECK: declare riscv_vls_cc(32768) void @riscv_vls_cc_32768()
 declare riscv_vls_cc(65536) void @riscv_vls_cc_65536()
 ; CHECK: declare riscv_vls_cc(65536) void @riscv_vls_cc_65536()
+declare cc124 void @f.cc124(i1)
+; CHECK: declare amdgpu_gfx_whole_wave void @f.cc124(i1)
```

Comment:
How does `cc124` become `amdgpu_gfx_whole_wave` here?

Reply:
That's because `amdgpu_gfx_whole_wave` has the enum value 124 (you can see the same kind of test in this file for our other calling conventions, e.g. `amdgpu_kernel` is `cc91` because we have `AMDGPU_KERNEL = 91` in `CallingConv.h`).